server/repository/src/db_diesel/requisition/requisition.rs

```diff
@@ -216,7 +218,24 @@ fn create_filtered_query(
         apply_string_filter!(query, comment, requisition_dsl::comment);
 
         apply_equal_filter!(query, store_id, requisition_dsl::store_id);
-        apply_equal_filter!(query, order_type, requisition_dsl::order_type)
+        apply_equal_filter!(query, order_type, requisition_dsl::order_type);
+
+        if let Some(shipment_created) = shipment_created {
+            let requisition_ids = invoice_dsl::invoice
```

Comment:
I think this can be simplified to just use invoice, don't need to join requisition

Reply:
[078ab88](https://github.com/msupply-foundation/open-msupply/pull/4385/commits/078ab886f9377578904887f19118e76681709a16)